Contest_API.md

```diff
@@ -1019,20 +1021,20 @@ organizations (i.e. return any organization\_ids).
 
 JSON elements of organization objects:
 
-| Name               | Type           | Required? | Nullable? | Description
-| :----------------- | :------------- | :-------- | :-------- | :----------
-| id                 | ID             | yes       | no        | Identifier of the organization.
-| icpc\_id           | string         | no        | yes       | External identifier from ICPC CMS.
-| name               | string         | yes       | no        | Short display name of the organization.
-| formal\_name       | string         | no        | yes       | Full organization name if too long for normal display purposes.
-| country            | string         | no        | yes       | [ISO 3166-1 alpha-3 code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) of the organization's country.
-| country_flag       | array of IMAGE | no        | yes       | Flag of the country. A server is recommended to provide flags of size around 56x56 and 160x160.
-| url                | string         | no        | yes       | URL to organization's website.
-| twitter\_hashtag   | string         | no        | yes       | Organization hashtag.
-| location           | object         | no        | yes       | JSON object as specified in the rows below.
-| location.latitude  | number         | depends   | no        | Latitude in degrees. Required iff location is present.
-| location.longitude | number         | depends   | no        | Longitude in degrees. Required iff location is present.
-| logo               | array of IMAGE | no        | yes       | Logo of the organization. A server must provide logos of size 56x56 and 160x160 but may provide other sizes as well.
+| Name               | Type          | Required? | Nullable? | Description
+| :----------------- | :------------ | :-------- | :-------- | :----------
+| id                 | ID            | yes       | no        | Identifier of the organization.
+| icpc\_id           | string        | no        | yes       | External identifier from ICPC CMS.
+| name               | string        | yes       | no        | Short display name of the organization.
+| formal\_name       | string        | no        | yes       | Full organization name if too long for normal display purposes.
+| country            | string        | no        | yes       | [ISO 3166-1 alpha-3 code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) of the organization's country.
+| country_flag       | array of FILE | no        | yes       | Flag of the country. A server is recommended to provide flags of size around 56x56 and 160x160. Only allowed mime types are image/*.
```

Comment:
I'm assuming that here and below the intention is that this is still restricted to the 3 image mime types we support: png, jpg and svg, right? That's still mentioned above but is now a little less explicit, but I think that's ok.

Reply:
> I'm assuming that here and below the intention is that this is still restricted to the 3 image mime types we support: png, jpg and svg, right? 

Yes.

> That's still mentioned above but is now a little less explicit, but I think that's ok.

I agree.